cmake/Find/FindCint.cmake

```diff
@@ -0,0 +1,41 @@
+
+#[=======================================================================[.rst:
+FindCint
+--------
+
+Find the Cint library (https://github.com/sunqm/libcint) on the system.
+
+Result Variables
+^^^^^^^^^^^^^^^^
+
+This module will set the following variables in your project:
```

Comment:
Could you also add to the documentation that a target for the Libcint library is made?